server/interfaces/overviewPage.ts

```diff
@@ -19,9 +19,17 @@ export interface OverviewSchedule {
 }
 
 export type OverviewNonAssociation = {
-  html?: string
-  text?: string
-}[]
+  nonAssociationName: string
+  offenderNo: string
+  assignedLivingUnitDescription: string
+  reasonDescription: string
+  agencyId: string
+}
+
+export type AlertsSummary = {
```

Comment:
Any specific reason to use a `type` here rather than `interface`?
We made an ADR to prefer `interface` over `type` unless there's a specific reason... https://github.com/ministryofjustice/hmpps-prisoner-profile/blob/main/architecture-decision-record/0002-prefer-interface-over-type.md

Reply:
No, will change. Just copied the pattern from the one above